examples/yoti_example_flask/requirements.in

```diff
@@ -1,9 +1,11 @@
-cffi>=1.14.0
+click>=7
+cffi>=1.15.0
 flask>=1.0.4
-jinja2>=2.8.1
+jinja2>=3.0.3
 pyopenssl>=19.0.0
 python-dotenv>=0.7.1
 requests>=2.20.0
 urllib3>=1.24.2
-yoti>=2.13.0
+yoti>=2.14.0
```

Comment:
Just a sense check: assume this is indeed correct as 2.14.**0** and does not need to match the Patch version (2.14.**1**) ?

Reply:
This won't actually compile/run until we release this new version. Generally we release the version, and then do the update to these, since they fail the CI checks otherwise :+1:

Reply:
(the CI checks make sure all the example projects compile succesfully)